app/assets/stylesheets/application.scss

```diff
@@ -13,6 +13,7 @@
 @import "components/index";
 @import "components/sidebar";
 @import "pages/index";
+@import "components/email";
```

Comment:
Could you move the "email" and "sidebar" to pages/index.scss?

Reply:
ok!

Reply:
Thank you. Will merge now!